docs/index.rst

```diff
@@ -3,17 +3,22 @@ Welcome to the Treeherder documentation!
 
 .. toctree::
    :maxdepth: 2
+   :caption: Getting Started
 
    installation
-   ui/installation
+   code_style
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Working with the Server
```

Comment:
Some of the items in this section are for things other than the server, eg building the docs, updating package.json, releasing a new python client, test cases. Perhaps "Common tasks" needs splitting up too? Happy to wait for another PR though if that's easier.

Also I'm not sure if mkdocs supports these captions (bug 1466084), but we can always cross that bridge then :-)

Reply:
@edmorley I've broken up the "Common tasks" section into two parts and put them under "Getting started". Also, I noticed a reference to buildbot - that should be removed, right?

Reply:
Fixed!